cmd/utils.go

```diff
@@ -93,19 +94,19 @@ func path2BucketObject(s string) (bucket, prefix string) {
 }
 
 func getDefaultParityBlocks(drive int) int {
-	return drive / 2
+	return int(math.Floor(float64(drive) / 2))
 }
 
 func getDefaultDataBlocks(drive int) int {
-	return drive - getDefaultParityBlocks(drive)
+	return int(math.Ceil(float64(drive) / 2))
```

Comment:
@harshavardhana Is reworking this AFAIK.

Reply:
This is already done for objects @klauspost what @vadmeste is adding this for buckets which didn't honor the same quorum ratio

In-fact here this is not correct all you have to do 

```suggestion
        return drive - getDefaultParityBlocks(drive)
```